Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -645,7 +644,11 @@ BlockHash branchpoint
 
             _chainDb.Write(batch);
 
-            long bpIndex = GetBlockIndex(branchpoint).Value;
+            long bpIndex = GetBlockIndex(branchpoint) is { } index
+                ? index
+                : throw new ArgumentException(
+                    $"Could not find block index with branchpoint '{branchpoint}' in store.",
+                    nameof(branchpoint));
```

Comment:
In general, `IStore`'s current overall design is not to throw an `Exception`. I think it'd be better to silently do nothing and terminate early.

Reply:
So is it right to return rather than throw an exception?
like below
```csharp
if (!(GetBlockIndex(branchpoint) is { } bpIndex))
{
    return;
}
```

Reply:
Whether it is "right" is a rather hard question to answer (although, I personally think that `IStore` have too much logic baked into it to be `Exception`-free design to be the right design; probably we need some kind of redesign for `IStore` at some point). Although the change would result in a behavioral change, we don't even really have the behavior of the method well-defined. 😑

Also, moving the check/early termination to an earlier point would be better, i.e. before `_chainDb.Write()` gets called for any modification.